es6-destructuring/index.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>ES6 Destructuring</title>
+</head>
+<body>
+  <script>
+   const book1 = {
+  title: 'Goodnight Punpun',
+  author: 'Inio Asano',
+  libraryID: 3353
+};
+const { title, author, libraryID } = book1;
+
+console.log(`The title of the book is ${title}, the author is ${author}, and the library id is ${libraryID}`);
+
+   const book2 = {
+     title: title2 = 'Les Fleurs du mal',
+     author: author2 = 'Charles Baudelaire',
+     libraryID: libraryID2 = 2345
+};
```

Comment:
`book2` should be defined in one statement and destructured in a different statement:

---

![image](https://user-images.githubusercontent.com/16143771/174714979-66f4663b-08f3-4453-9177-977671afbed2.png)

Reply:
Are you looking for something like this?

![image](https://user-images.githubusercontent.com/97073517/174715979-9d7e757d-2e01-4f91-bb56-1b576e1061dd.png)

Reply:
@maryalenn It should be destructuring, but `book2` should be an object (as in the screenshot of the instructions), not an array.